src/constants.js

```diff
@@ -0,0 +1,39 @@
+export const stateCodes = {
+	'Andhra Pradesh': 'AP',
+	'Arunachal Pradesh': 'AR',
+	'Assam': 'AS',
+	'Bihar': 'BR',
+	'Chhattisgarh': 'CT',
+	'Goa': 'GA',
+	'Gujarat': 'GJ',
+	'Haryana': 'HR',
+	'Himachal Pradesh': 'HP',
+	'Jharkhand': 'JH',
+	'Karnataka': 'KA',
+	'Kerala': 'KL',
+	'Madhya Pradesh': 'MP',
+	'Maharashtra': 'MH',
+	'Manipur': 'MN',
+	'Meghalaya': 'ML',
+	'Mizoram': 'MZ',
+	'Nagaland': 'NL',
+	'Odisha': 'OR',
+	'Punjab': 'PB',
+	'Rajasthan': 'RJ',
+	'Sikkim': 'SK',
+	'Tamil Nadu': 'TN',
+	'Telangana': 'TG',
+	'Tripura': 'TR',
+	'Uttarakhand': 'UT',
+	'Uttar Pradesh': 'UP',
+	'West Bengal': 'WB',
+	'Andaman and Nicobar Islands': 'AN',
+	'Chandigarh': 'CH',
+	'Dadra and Nagar Haveli': 'DN',
+	'Daman and Diu': 'DD',
```

Comment:
This wont be needed here once I create a backend API.

Reply:
https://api.livecovid.in/api/states/testing

Use this API to get daily testing data for each state.

Reply:
Yes.